README.adoc

```diff
@@ -43,8 +41,33 @@ If you already have Rust installed, make sure you're using the latest version by
 rustup update
 ----
 
+=== Install PoC-4 on "Alexander" Testnet
+
+Build Polkadot PoC-4 by cloning this repository and running the following commands from the root directory of the repo:
+
+```bash
+git checkout v0.4
+./scripts/init.sh
+./scripts/build.sh
+cargo build --release
```

Comment:
this isn't entirely consistent with the w3f wiki https://wiki.polkadot.network/en/latest/polkadot/node/guides/how-to-validate/#install-polkadot-poc-4